vunit/sim_if/_ossmixin.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2014-2024, Lars Asplund lars.anders.asplund@gmail.com
+"""
+Viewer handling for GHDL and NVC.
+"""
+
+
+class OSSMixin:
```

Comment:
Open source software. But wasn't very happy with that to renamed to ViewerMixin.

(I had some idea to merge more of the NVC and GHDL classes, but since it now ended up to be viewer only...)